src/pybrake/notifier.py

```diff
@@ -116,6 +121,15 @@ def __init__(
         self._context = _CONTEXT.copy()
         self._context["rootDirectory"] = kwargs.get("root_directory",
                                                     os.getcwd())
+        self._failed_stats = None
+        if self.config.get('manage_failed_stats'):
+            self._failed_stats = FailedStats(
+                interval=FLUSH_PERIOD,
+                header=self._ab_headers,
+                url=self._ab_url,
+                method="POST",
```

Comment:
Is it important to make `method` configurable?

Reply:
Method used in the `send` function, It has already been configured. However, because this API is a POST call, I passed hard coded over here.